### PR TITLE
Docs: Add 7.5 What's New links

### PR DESCRIPTION
**Pull request by oddlittlebird**
Added links to updated documentation in What's new in Grafana 7.5.

Minor updates in Postgres and Azure Monitor data source topics to improve title consistency and remove outdated information.

### CODE REVIEW

**Review by wbrowne — CHANGES_REQUESTED** [review]
Since we're removing the alias here, we'll need to update the following to the correct link:

```
"grafana": {
    "whatsNewUrl": "https://grafana.com/docs/grafana/latest/guides/whats-new-in-v7-5/",
 ...
```

inside the root `package.json`. And therefore this will need to be backported to the 7.5.x  release branch so that it goes out with the beta2+stable. Looping in @Elfo404 to confirm my thinking here.

**Comment by Elfo404** [comment]
@wbrowne I'd say what you say is correct.

important: if we remove the alias we also need to make sure the link in https://grafana.com/grafana/download/7.5.0-beta1 is updated correctly (we can use `gcom` for this)

File: docs/sources/datasources/postgres.md [hunk 1]
```diff
@@ -6,18 +6,13 @@ aliases = ["/docs/grafana/latest/features/datasources/postgres/"]
 weight = 1200
 +++
 
-# Using PostgreSQL in Grafana
+# PostgreSQL data source
 
-Grafana ships with a built-in PostgreSQL data source plugin that allows you to query and visualize data from a PostgreSQL compatible database.
+Grafana ships with a built-in PostgreSQL data source plugin that allows you to query and visualize data from a PostgreSQL compatible database. This topic explains options, variables, querying, and other options specific to the this data source. Refer to [Add a data source]({{< relref "add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.
```
Comment (osg-grafana) [inline docs/sources/datasources/postgres.md:11]:
```suggestion
Grafana ships with a built-in PostgreSQL data source plugin that allows you to query and visualize data from a PostgreSQL compatible database. This topic explains options, variables, querying, and other options specific to this data source. For instructions about how to add a data source to Grafana, refer to [Add a data source]({{< relref "add-a-data-source.md" >}}). Only users with the organization admin role can add data sources.
```

**Comment by oddlittlebird** [comment]
I added the alias back in for now, but could someone please fix the incorrect grafana.com link?

**Comment by oddlittlebird** [comment]
Could someone please approve this?

**Review by achatterjee-grafana — APPROVED** [review]

**Comment by oddlittlebird** [comment]
@Elfo404 and @wbrowne, what is the status of this? Are we good to merge?

**Review by wbrowne — APPROVED** [review]
Yep, no alias change so all good @oddlittlebird!

**Pull request merged**